clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -1411,7 +1417,7 @@ void SYCLToolChain::AddImpliedTargetArgs(const llvm::Triple &Triple,
                      options::OPT_fno_target_export_symbols, false))
       BeArgs.push_back("-library-compilation");
   } else if (Triple.getSubArch() == llvm::Triple::NoSubArch &&
-             Triple.isSPIR()) {
+             (Triple.isSPIR() || Triple.isSPIRV())) {
```

Comment:
This check is showing up quite a bit (spir/spirv and no subarch) does it make sense to simplify this too?

Reply:
I would like to leave this as is. I wanted to NOT add too many helper functions in Triple.h and bloat it up. Adding it here would change the signature. I will try to attempt it when we clean up spir/spir64.

Hope that's ok.
Thanks